boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -201,8 +211,18 @@ gdt:
 
 /* GDT table pointer */
 gdtptr:
-    .word HEX(17)   /* Limit */
-    .long gdt       /* Base Address */
+    .word HEX(17)       /* Limit */
+#ifdef _USE_ML
+    .long offset gdt    /* Base Address */
```

Comment:
Doesn't this work for both ML and GAS?

Reply:
it works, but I saw this in arch/realmode/amd64.S and so I decided to put.

Reply:
If offset works for both and generates the same code, lt's just use that.

Reply:
i reverted this because it's useless.

Reply:
I reverted!